src/kvstore/plugins/cortx/cortx_kvstore.c

```diff
@@ -24,45 +24,57 @@
  
 #include "internal/cortx/cortx_kvstore.h"
 #include <cortx/helpers.h>
+#include "operation.h"
 
 int cortx_kvs_init(struct collection_item *cfg_items)
 {
+    perfc_trace_inii(PFT_CORTX_KVS_INIT, PEM_NSAL_TO_MOTR);
 	return m0init(cfg_items);
+    perfc_trace_finii(PERFC_TLS_POP_DONT_VERIFY);
```

Comment:
This line will never execute, this is a bug.

Reply:
fixed.